tests/functional/test_override_ssl_common_name.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+from awscli.testutils import mock, create_clidriver
+import pytest
+
+from awscli.customizations.overridesslcommonname import ssl_common_names
+
+regions_to_exclude = [
+    "af-south-1",
+    "ap-east-1",
+    "ap-southeast-3",
+    "eu-south-1",
+    "me-south-1",
+    "cn-north-1",
+    "cn-northwest-1",
+    "us-gov-west-1",
+    "us-isob-east-1",
+]
+sqs = [
+    ("sqs", "list-queues", region)
+    for region in ssl_common_names["sqs"]
+    if region not in regions_to_exclude
+]
+emr = [
+    ("emr", "list-clusters", region)
+    for region in ssl_common_names["emr"]
+    if region not in regions_to_exclude
+]
+shield = [
+    ("shield", "list-protections", region) for region in ssl_common_names["shield"]
+]
+rds = [("rds", "describe-db-clusters", region) for region in ssl_common_names["rds"]]
+test_cases = sqs + emr + shield + rds
+
+driver = create_clidriver()
```

Comment:
Can we make this a [fixture](https://docs.pytest.org/en/6.2.x/fixture.html) so it can be shared among tests without any concerns for parallelization?

Reply:
will do!